### PR TITLE
[WIP] add drone

### PR DESCRIPTION
**Pull request by felixmulder**
Drone is our new CI infrastructure based on docker images instead of VMs in Jenkins - the address is http://dotty-ci.epfl.ch. All members of the lampepfl group have access to the internals of this site.

### CODE REVIEW

**Comment by smarter** [comment]
> [error] (dotty-interfaces/compile:doc) java.io.IOException: Cannot run program "javadoc": error=2, No such file or directory

You need javadoc in your docker image I guess

**Comment by felixmulder** [comment]
It seems to work fine now, I'm merging this so that we can have both CIs running at the same time to try out the differences.

This will also help @nicolasstucki - since everything above 100 commits gets ignored by the other CI.

**Comment by felixmulder** [comment]
Also - note that the new CI will re-run tests on master to see if anything broke after merge.

File: compiler/test/dotty/partest/DPConsoleRunner.scala [hunk 1]
```diff
@@ -105,9 +105,9 @@ extends SuiteRunner(testSourcePath, fileManager, updateCheck, failed, javaCmdPat
     val seqResults =
       if (!sequentialTests.isEmpty) {
         val savedThreads = sys.props("partest.threads")
-        sys.props("partest.threads") = "1"
+        sys.props("partest.threads") = "2"
 
-        NestUI.echo(s"## we will run ${sequentialTests.length} tests sequentially")
+        NestUI.echo(s"## we will run ${sequentialTests.length} tests using ${PartestDefaults.numThreads} thread(s)")
```
Comment (smarter) [inline compiler/test/dotty/partest/DPConsoleRunner.scala]: You did not update the documentation and the variable names to reflect the fact that these tests are not run sequentially anymore.

Comment (felixmulder) [inline compiler/test/dotty/partest/DPConsoleRunner.scala]: d'oh! On it

**Pull request merged**